kedro/framework/cli/starters.py

```diff
@@ -93,6 +93,24 @@ class KedroStarterSpec:  # noqa: too-few-public-methods
     "An optional directory inside the repository where the starter resides."
 )
 
+# TODO; Insert actual link to the documentation.
+ADDON_ARG_HELP = """
+Select which add-ons you'd like to include. By default, none are included.
+Visit: kedro.org/{insert-documentation} to find out more about these add-ons.
+
+Add-Ons
+1) Linting : Provides a basic linting setup with Black and Ruff
+2) Testing : Provides basic testing setup with pytest
+3) Custom Logging : Provides more logging options
+4) Documentation: Basic documentation setup with Sphinx
+5) Data Structure: Provides a directory structure for storing data
+
+Example usage:
+kedro new --addons=lint,test,log,docs,data (or any subset of these options)
+kedro new --addons=all
+kedro new --addons=none
```

Comment:
To have this print like this when you do `kedro new --help` you'll need to add `\n` at the end of every line to get a new line in between, e.g:

```
Add-Ons \n
1) Linting: Provides a basic linting setup with Black and Ruff \n
2) Testing: Provides basic testing setup with pytest \n
``` 

etc..

Reply:
For this text block, I've tried adding the line breaks before, and the result is a bit strange. It adds a larger break between lines, like this:

![image](https://github.com/kedro-org/kedro/assets/57910428/3d58fcc8-0be9-4b1e-8cb0-5807508a00d4)

I've also tried using `\b` instead of `\n`, and formatting it with methods like `click.wrap_text()`, but none of that gives me the formatting I actually want. I'll leave it with the `\n`s for now because it's easier to read.